### ISSUE TITLE
Accessibility: Link element under the chart (keyboard accessibility, contrast, role)

### ISSUE THREAD

**Issue by rezinak** [highcharts/highcharts#18516]
#### Expected behaviour
* Highcharts.com link under the chart is focusable with the keyboard and the focus state of the element is visually indicated ([WCAG criteria for keyboard accessibility](https://www.w3.org/WAI/WCAG22/Understanding/keyboard-accessible) and [focus visibility](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html) )
* Highcharts.com link is announced as a "link" by screen readers to fulfill [Name, role, value WCAG criteria](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)
* The text has a sufficient contrast to fulfill [WCAG criteria](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html). Target contrast for the element is 4.5:1

#### Actual behaviour
* The link is not keyboard-accessible. Keyboard-only users will be prevented from accessing the link.
* Highcharts.com element does not announce role (link). Screen reader users will have difficulty determining the purpose and state of these controls.
* The element doesn't have WCAG-required contrast ratio. Users with low vision will have difficulty reading this content. Current contrast ratio is 2.849:1

#### Live demo with steps to reproduce
https://www.idc.com/QA_test_section/accessibility-remediation/highcharts-demo1 

#### Product version
Highcharts JS v10.3.3 

#### Affected browser(s)
Chrome and Mozilla, haven't looked into the others

**Comment by oysteinmoseng** [comment]
Contrast workaround: The color can be configured manually using the [credits.style.color](https://api.highcharts.com/highcharts/credits.style.color) option.

Internal note: Why is this not an SVG <a> element instead of <text>? Would be the simple solution.

**Comment by rezinak** [comment]
> Contrast workaround: The color can be configured manually using the [credits.style.color](https://api.highcharts.com/highcharts/credits.style.color) option.
> 

Thank you. Do you know if the highcharts.com link can be removed from the page altogether? Maybe in the paid version?
